commands/vehicle.go

```diff
@@ -79,6 +90,19 @@ func RequestPIDRaw(unitID uuid.UUID, name, headerHex, modeHex, pidHex string, pr
 
 	cmd := fmt.Sprintf(`%s %s header="'%s'" mode='x%s' pid='x%s' protocol=%d force=true`,
 		api.ObdPIDQueryCommand, name, headerHex, modeHex, pidHex, protocol)
+
+	if strings.Contains(request.Formula, "python") {
```

Comment:
the formula will always be like eg. `python: bytes_to_int(messages[0].data[-1:])*0.5` so you could do request.formula startswith "python:", and then when appending you'll want to remove the "python:" portion from the beginning.

Reply:
Good catch, updated